src/libs/Middleware/Logging.ts

```diff
@@ -32,15 +32,25 @@ function logRequestDetails(message: string, request: Request, response?: Respons
         logParams.requestID = response.requestID;
     }
 
-    Log.info(message, false, logParams);
+    const extraData: Record<string, unknown> = {};
+    /**
+     * We don't want to log the request and response data for AuthenticatePusher
+     * requests because they contain sensitive information.
+     */
+    if (request.command !== 'AuthenticatePusher') {
+        extraData.request = request;
```

Comment:
This change raise regression issue #47509 when enable client log request data include has file can not store to indexdb, so we should serialize request data